src/main/resources/jenkins/branch/DescriptionColumn/column.jelly

```diff
@@ -4,10 +4,10 @@
     <j:set var="p" value="${col.getPropertyOf(job)}"/>
     <j:choose>
       <j:when test="${p != null and p.objectUrl != null}">
-        <a href="${p.objectUrl}">${col.description(p,job)}</a>
+        <a href="${p.objectUrl}"><j:out value="${col.description(p,job)}"/></a>
       </j:when>
       <j:otherwise>
-        ${col.description(p, job)}
+        <j:out value="${col.description(p, job)}"/>
```

Comment:
Ok, I asked around, and normally you would need an explicit call to `MarkupFormatter.translate` like [this](https://github.com/jenkinsci/jenkins/blob/22aa2e6e766074d11249893e3f35e0b99e20d3d0/core/src/main/resources/hudson/widgets/HistoryWidget/entry.jelly#L72) or [this](https://github.com/jenkinsci/credentials-plugin/blob/6f710a7e30156e94aff8a3c6c6799cef04ad35df/src/main/resources/com/cloudbees/plugins/credentials/CredentialsStoreAction/DomainWrapper/index.jelly#L33). However, in this case, `DescriptionColumn.description(Object, Object)` does that call internally: https://github.com/jenkinsci/branch-api-plugin/blob/db3c6487e3fa9f0214b6e50e5808eba84b531aff/src/main/java/jenkins/branch/DescriptionColumn.java#L95

So I think that makes this change safe as-is. Probably worth a comment here explaining why, since normally it would not be. Maybe clearer to make the Java code return the raw content and do the escaping in Jelly, although you'd need to make sure other uses of `DescriptionColumn.description(Object, Object)` didn't have any issues and that escaping the result of the first branch from the Java code doesn't break anything.

Reply:
With this return value, the job description gets translated into HTML according to the selected markup formatter. In OWASP Markup Formatter that's done in [RawHtmlMarkupFormatter.translate()](https://github.com/jenkinsci/antisamy-markup-formatter-plugin/blob/c21daf9debe89aa45314ede7cf25f6f734ab3002/src/main/java/hudson/markup/RawHtmlMarkupFormatter.java#L33-L52), using the [MyspacePolicy](https://github.com/jenkinsci/antisamy-markup-formatter-plugin/blob/c21daf9debe89aa45314ede7cf25f6f734ab3002/src/main/java/hudson/markup/MyspacePolicy.java).

The default markup formatter in the Jenkins core however is using the [plain text markup formatter](https://github.com/jenkinsci/jenkins/blob/a5db82c0253dc88d72aa13ebeab225f0db2aa46c/core/src/main/java/hudson/markup/EscapedMarkupFormatter.java#L49-L51), that escapes the syntax by the rules defined in the [Util class](https://github.com/jenkinsci/jenkins/blob/a5db82c0253dc88d72aa13ebeab225f0db2aa46c/core/src/main/java/hudson/Util.java#L908-L946).